### PR TITLE
 ASCOM latex as a paper style

### PR DESCRIPTION
**Pull request by womullan**
requires the latest lsst-texmf to generate the authors.tex

### CODE REVIEW

File: project_templates/technote_spietex/testn-000/authors.tex [hunk 1]
```diff
@@ -1,8 +1,7 @@
 %% DO NOT EDIT THIS FILE. IT IS GENERATED FROM db2authors.py"
 %% Regenerate using:
-%%    python $LSST_TEXMF_DIR/bin/db2authors.py > authors.tex
+%%    python $LSST_TEXMF_DIR/bin/db2authors.py Namespace(mode='spie', noafil=False)
```
Comment (jonathansick) [inline project_templates/technote_spietex/testn-000/authors.tex:3]: Do you know what's going on here with the generated argument to `db2authors.py`? Looks like you're printing a dataclass?

**Review by jonathansick — CHANGES_REQUESTED** [review]
Thanks. One thing I noticed is that all the "dot files" are missing. I'd expect to see a `.gitignore` and a `.github/workflows/ci.yaml`-type file.

File: project_templates/technote_spietex/testn-000/authors.tex (hunk 1 above)
Comment (womullan) [reply project_templates/technote_spietex/testn-000/authors.tex:3]: oh i had not spoted that - i will check and add the missing .files

Comment (womullan) [reply project_templates/technote_spietex/testn-000/authors.tex:3]: previously i was not passing arguments -- in the script i print {args} i fixed that

**Comment by womullan** [comment]
I think that covers the comments

**Review by jonathansick — APPROVED** [review]
Thanks I think this will work for us.

**Pull request merged**